JoyboyCommunity/src/modules/Tip/styles.ts

```diff
@@ -0,0 +1,65 @@
+import {Platform} from 'react-native';
+
+import {Spacing, ThemedStyleSheet} from '../../styles';
+
+export default ThemedStyleSheet((theme) => ({
+  modal: {
+    backgroundColor: theme.colors.surface,
+    paddingLeft: Spacing.medium,
+    paddingRight: Spacing.medium,
+    paddingBottom: Spacing.medium,
+    height: 728,
+  },
+
+  header: {
+    width: '100%',
+    marginBottom: Spacing.medium,
+    paddingTop: Spacing.small,
+    paddingLeft: Spacing.small,
+    paddingRight: Spacing.small,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  icon: {
+    color: theme.colors.primary,
+  },
+
+  postHeader: {flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center'},
+
+  innerPostHeader: {flexDirection: 'row', gap: 5, alignItems: 'center'},
+  title: {
+    marginBottom: Spacing.xsmall,
+  },
+  likes: {flexDirection: 'row', gap: 3, alignItems: 'center'},
+
+  recipient: {flexDirection: 'row', gap: 4, alignItems: 'center', paddingBottom: 10},
+  post: {
+    width: '100%',
+    height: 112,
+    backgroundColor: '#EC796B1A',
```

Comment:
This color already defined in the theme if im not wrong, it should be primaryLight